main.py

```diff
@@ -128,8 +128,14 @@ def handle_enable_table(self, event):
         """Handle NApps responses from enable_table
         Second, wait for all the napps to respond"""
         napp = event.name.split("/")[1].split(".")[0]
-        # Check against the last current table
-        self.required_napps.remove(napp)
+        try:
+            self.required_napps.remove(napp)
+        except KeyError:
+            log.error(
+                f"{napp} NApp loaded after pipeline was installed."
+                " Flow inconsistencies may have appeared."
```

Comment:
Should it also try to tell the user what to do? By making sure NApps were installed and loaded in the correct order and restart the controller and/or making sure the depending `SUBSCRIBED_NAPPS` NApps are also enabled?

Reply:
Modified log to:
```
                f"{napp} NApp loaded after pipeline was installed. "
                "Flow inconsistencies may have appeared. "
                f"Try modifying `kytos.json` from {napp} then restart or 
                redeploy the pipeline."
```